runtime/error.c

```diff
@@ -1,12 +1,128 @@
+#include <string.h>
+#include <stdlib.h>
+#include <stdio.h>
+
 #include "balrt.h"
+#include "third-party/libbacktrace/backtrace.h"
+
+#define INITIAL_PC_COUNT 1
+#define MAX_PC_COUNT 256
+#define SKIP_FROM_END 1 // This number counts from the last stack frame
+#define THREAD 0
+#define SUCCESS 0
+#define FAIL 1
+
+struct backtrace_state *state = NULL;
+
+typedef struct {
+    uint32_t nPCs;
+    uint32_t szPCs;
+    bool errorOccurred;
+    PC *pcs;
+} PCVector;
```

Comment:
Now that you have added errorOccurred (which is a good addition), PCVector is no longer a good name. PCVector is now saving the result of  a call to backtrace_simple. So I think I woud rename it to `SimpleBacktrace`.

Reply:
Fixed